helm_drydock/ingester/__init__.py

```diff
@@ -31,55 +32,62 @@
 class Ingester(object):
 
     def __init__(self):
-        logging.basicConfig(format="%(asctime)-15s [%(levelname)] %(module)s %(process)d %(message)s")
-        self.log = logging.Logger("ingester")
+        self.logger = logging.getLogger("drydock.ingester")
         self.registered_plugins = {}
 
     def enable_plugins(self, plugins=[]):
```

Comment:
Initializing plugins=[] is a "common gotcha"

Better to do plugins=None

```python

if plugins is None:
    plugins = []
```

More info: http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/